ietf/settings.py

```diff
@@ -809,8 +809,8 @@ def skip_unreadable_post(record):
 SESSION_REQUEST_FROM_EMAIL = 'IETF Meeting Session Request Tool <session-request@ietf.org>' 
 
 SECRETARIAT_SUPPORT_EMAIL = "support@ietf.org"
-SECRETARIAT_ACTION_EMAIL = "ietf-action@ietf.org"
-SECRETARIAT_INFO_EMAIL = "ietf-info@ietf.org"
+SECRETARIAT_ACTION_EMAIL = SECRETARIAT_SUPPORT_EMAIL
```

Comment:
What were the replaced addresses doing? Old aliases in the mail system?

Reply:
the secretariat used to use these aliases to drop tickets into separate queues for different types of requests. Now, everything is intended to go support@ instead, with queuing happening later, so that the community doesn't have to figure out which of many addresses to use. We have code that could be refactored to simplify having this many settings, but that should be a separate PR.

Reply:
also - the other aliases just forward to support@ currently